docs/content/Coding Milestones/PoC2023q3/outline.md

```diff
@@ -2,4 +2,78 @@
 title: "Details"
 ---
 
-# Coming Soon
+Want to get involved? Check out our [good-first-issue list]({{ config.repo_url }}/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+TODO: draw new picture
+
+## Status of this memo
+
+This document outlines near-term plans for building on what was
+produced in the work on [the PoC for 2023q1](../../PoC2023q1/outline/).
+
+## Introduction
+
+PoC2023q1 was defined with the over-arching goal of supporting edge
+computing scenarios.  Since then we have realized that the technical
+problems that we took on are not that specific, they appear in other
+multi-cluster scenarios as well.
+
+The goals of this PoC are as follows.  Ones that are substantially
+different from what has been accomplished for PoC2023q1 are
+highlighted.
+
+- Separation of infrastructure and workload management.
+- The focus here is on workload management, and that strictly reads
+  an inventory of infrastructure.
+- What passes from inventory to workload management is kcp TMC
+  Location and SyncTarget objects.
+- **Compared to PoC2023q1, decoupling from kcp TMC by making our own
+  copy of the definitions of SyncTarget and Location.**
+- **Potentialy: switch from using SyncTarget and Location to some
+  other representatin of inventory.**
```

Comment:
Some typos (potentially, representation) in this bullet point?

Reply:
Good catch, thanks.
@clubanderson, I thought we had spell checking that should have caught this?